openapi_tester/schema_tester.py

```diff
@@ -90,17 +91,17 @@ def handle_one_of(
             )
 
     @staticmethod
-    def _check_openapi_type(schema_type: str, value: Any, enum: Optional[List[Any]], format: Optional[str]) -> bool:
+    def _check_openapi_type(schema_type: str, value: Any, enum: Optional[List[Any]], _format: Optional[str]) -> bool:
         if enum:
             return value in enum
         if schema_type == "boolean":
             return isinstance(value, bool)
         if schema_type in ["string", "file"]:
-            if format == "bytes":
+            if _format == "byte":
```

Comment:
Looks like the correct format is `byte` over `bytes` :) 

https://swagger.io/docs/specification/data-models/data-types/#string

Reply:
why the variable name change?

Reply:
just so we're not overwriting the `format` builtin. 

It shouldn't matter for the code in this repo, but I think there's a risk that we can change behavior in downstream code by overwriting it.

https://www.programiz.com/python-programming/methods/built-in/format

Reply:
I guess it might not matter since we're in the scope of the single function, but also don't see a very good reason to use builtin namespaces if we can avoid it

Reply:
ok, in general i prefer to use _ for private class methods and private variables, for function scope variables, just informative naming. So this could be schema_format or format_type etc. I also do not think we have a problem with the builtin spacename - its a safe use case.